updater.js

```diff
@@ -0,0 +1,252 @@
+var path = require('path')
+var fs = require('fs')
+var childProcess = require('child_process')
+var pkgUp = require('pkg-up')
+
+var BrowserslistError = require('./error')
+
+var UNKNOWN_VERSION = 'unknown version'
+var PACKAGE_CANIUSE = 'caniuse-lite'
+
+var packageManager = ''
```

Comment:
Using a global state is not good for simple functions like we have in this task.

The code will be more readable if we will pass it to functions as an argument.